src/components/ItemList/ItemList.js

```diff
@@ -0,0 +1 @@
+ItemList
```

Comment:
Codacy found an issue: ['ItemList' is not defined.](https://app.codacy.com/gh/WebHeroSchool/evafinal/pullRequest?prid=5874309)